template.yml

```diff
@@ -394,6 +412,21 @@ Parameters:
     Type: String
     Description: This is used to force an update of the config if no other changes are made
     Default: "1"
+  DeployedRegions:
+    Type: AWS::SSM::Parameter::Value<String>
+    Default: /<EnvironmentName>/nucleus/deployedRegions
+  DdbGlobalTableArn:
+    Type: AWS::SSM::Parameter::Value<String>
+    Default: /<EnvironmentName>/nucleus/core/customResources/ddbGlobalTable
+
+Conditions:
+  IsSecondaryRegion: !Not
```

Comment:
this isn't something I knew when I started writing these, you can reference a condition in here: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-condition.html
so it could be something like `IsSecondaryRegion: !Not [{Condition: IsPrimary}]`

Reply:
maybe not the best place to test it since we can't deploy this one right now.

Reply:
Done.
Used `IsSecondaryRegion: !Not [!Condition IsPrimary]`